settings.py

```diff
@@ -26,6 +26,9 @@ def validate(cls, value: str) -> str:
 
 class Settings(BaseSettings):
     secret_key: str
+    azure_client_id: str
+    azure_client_secret: str
+    azure_tenant: str
```

Comment:
Very minor comment, but I'd prefer if these settings were visually separated from `secret_key`. I'd like to keep the settings grouped together with blank lines between each group to hopefully assist people's understanding of what these settings do.